src/plugins/discover/public/embeddable/saved_search_embeddable.test.ts

```diff
@@ -72,6 +72,7 @@ describe('saved search embeddable', () => {
       columns: ['message', 'extension'],
       rowHeight: 30,
       rowsPerPage: 50,
+      title: 'custom title',
```

Comment:
Do we have another test without a custom title?

Reply:
Thx, you're right, we should do it differently, I've refactored, added another test, wanna have another look?

Reply:
Looking good, thanks!